.github/workflows/ci-format.yml

```diff
@@ -58,13 +58,18 @@ jobs:
           . ${{env.venv_dir}}/bin/activate
           pip install --upgrade pip wheel
           pip install .[camera-obs,dev,extras,opendrive,ros,test,train]
-      - name: Get changed files
+          # install the specific version we want for pytest, just in case...
+          pip install pytype==2022.1.13
+      - name: Get changed files on branch since branching
         id: changed-files
-        uses: tj-actions/changed-files@v13.2  
-        with:
-          base_sha: origin/develop     
+        shell: bash
+        run: |
+          cd $GITHUB_WORKSPACE
+          CHANGED=$(git diff --diff-filter="AM" --name-only --ignore-submodules=all origin/develop... | awk -v d=" " '{s=(NR==1?s:s d)$0}END{print s}')
```

Comment:
Interesting, I would guess the ellipsis after a branch means the direct parent node?

Reply:
The "unfinished ellipsis" is just short for `...HEAD`, i.e., comparing to the head of the current branch.

The key reason for using the triple-dot ellipsis here is so we compare the PR branch to `develop` "at the branch point".  I.e., we are ignoring anything that's happened on the `develop` branch since we branched, so the only files that should show up in this diff are things that were changed in the PR, _not_ things that were changed in the meantime on `develop`.